web-client/Cargo.toml

```diff
@@ -46,6 +46,7 @@ nimiq-primitives = { workspace = true, features = ["coin", "networks", "ts-types
 nimiq-serde = { workspace = true }
 nimiq-transaction = { workspace = true, features = ["ts-types"] }
 nimiq-transaction-builder = { workspace = true }
+nimiq-utils = { workspace = true, features = ["merkle"]}
```

Comment:
```suggestion
nimiq-utils = { workspace = true, features = ["merkle"] }
```

Reply:
Interesting that `Cargo.toml` files are not covered by `cargo fmt`.